src/genie/libs/parser/iosxe/tests/ShowVrrp/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,41 @@
+expected_output = { 
+     {
+    "interface": {
+        "GigabitEthernet1": {
+            "group_number": {
+                "Group 100": {
+                    "auth_text": '"hash"',
+                    "device_advertise_interval": "1.000",
+                    "master_adv_int": "1.000",
+                    "master_down_int": "3.531",
+                    "master_ip_address": "192.168.1.233",
+                    "master_router": "local",
+                    "master_router_priority": "120",
+                    "preemption_state": "enabled",
```

Comment:
idea is, as long as it makes sense, better to use key name based on output. In this case,
```suggestion
                    "preemption": "enabled",
```
corresponding output:
```
  Preemption enabled
```

Reply:
confirmed the change.